js/render_entries.js

```diff
@@ -140,52 +168,97 @@ function _render_tests_table(uid, tests, earlier_tests_map) {
 
     if (test.hasOwnProperty("reason")) {
       if (text)
-    text += '<br>';
-
-    text += test.reason;
+        text += '<br>';
+      text += test.reason;
     }
 
     raw_html += '<tr class="' + row_class + '">';
     raw_html += '<td>' + test.name + '</td>';
     raw_html += '<td>' + test.result + '</td>';
 
-    raw_html += '<td class="mono">';
-    if (test.hasOwnProperty("memory"))
-      raw_html += test.memory + " B";
+    if (test.hasOwnProperty('memstat')) {
+      if (test.memstat['heap-jerry'] != 'n/a') {
+        raw_html += '<td class="mono">' + test.memstat['heap-jerry'] + ' B</td>';
+      } else {
+        raw_html += '<td class="mono">' + test.memstat['heap-jerry'] + '</td>';
+      }
```

Comment:
Maybe we can write this block in the following format:
```js
var unit = test.memstat['heap-jerry'] != 'n/a' ? ' B' : '';
raw_html += '<td class="mono">' + test.memstat['heap-jerry'] + unit + '</td>';
```
Or it deserves a small helper function.